moved/src/types/state.rs

```diff
@@ -111,34 +111,37 @@ impl From<ExtendedBlock> for BlockResponse {
         Self {
             header: alloy::rpc::types::Header {
                 hash: value.hash,
-                number: value.block.header.number,
-                parent_hash: value.block.header.parent_hash,
-                uncles_hash: value.block.header.ommers_hash,
-                nonce: Some(value.block.header.nonce.into()),
-                base_fee_per_gas: Some(value.block.header.base_fee_per_gas.to_saturated_u64()),
-                blob_gas_used: Some(value.block.header.blob_gas_used),
-                excess_blob_gas: Some(value.block.header.excess_blob_gas),
-                parent_beacon_block_root: Some(value.block.header.parent_beacon_block_root),
-                logs_bloom: value.block.header.logs_bloom.into(),
-                transactions_root: value.block.header.transactions_root,
-                state_root: value.block.header.state_root,
-                receipts_root: value.block.header.receipts_root,
-                difficulty: value.block.header.difficulty,
-                extra_data: value.block.header.extra_data,
-                gas_limit: value.block.header.gas_limit,
-                gas_used: value.block.header.gas_used,
-                timestamp: value.block.header.timestamp,
+                inner: alloy::consensus::Header {
+                    number: value.block.header.number,
+                    parent_hash: value.block.header.parent_hash,
+                    ommers_hash: value.block.header.ommers_hash,
+                    nonce: value.block.header.nonce.into(),
+                    base_fee_per_gas: Some(value.block.header.base_fee_per_gas.to_saturated_u64()),
+                    blob_gas_used: Some(value.block.header.blob_gas_used),
+                    excess_blob_gas: Some(value.block.header.excess_blob_gas),
+                    parent_beacon_block_root: Some(value.block.header.parent_beacon_block_root),
+                    logs_bloom: value.block.header.logs_bloom.into(),
+                    transactions_root: value.block.header.transactions_root,
+                    state_root: value.block.header.state_root,
+                    receipts_root: value.block.header.receipts_root,
+                    difficulty: value.block.header.difficulty,
+                    extra_data: value.block.header.extra_data,
+                    gas_limit: value.block.header.gas_limit,
+                    gas_used: value.block.header.gas_used,
+                    timestamp: value.block.header.timestamp,
+                    beneficiary: value.block.header.beneficiary,
+                    // TODO: review fields below
```

Comment:
Just curious, can we review this `TODO` by calling the integration test?

Reply:
Yeah, the integration test might help by letting us look at real, fully populated values coming from `op-geth`. We should also review the documentation to check our understanding of what the fields mean.